test/channels.test.js

```diff
@@ -605,4 +605,68 @@ describe.only('Workshop\'s event propagation into channels', () => {
 
         await workshops.patch(workshopNew.id, { xml: '<xml />' });
     });
+
+    it('does receive properties-first-user-changed', async () => {
```

Comment:
It looks like these tests will work properly, however I'm missing a clear assertion parts. The assertions are not only checking the specific conditions that confirm the validity of code but also provide reader with clear information what this condition is.

But please do not bother fixing it now. It's a comment for your future tests and (if there is time for it) for a refactor session.